src/particles/feedback_CIC_gpu.cu

```diff
@@ -73,6 +132,15 @@ void supernova::initState(struct parameters *P, part_int_t n_local, Real allocat
 }
 
 
+__device__ Real GetSNRate(Real t, Real* d_snr, Real snr_dt, Real t_start, Real t_end) {
```

Comment:
should this be in the `supernova` namespace as well?

Reply:
I don't expect this function to be used outside this file, so didn't think to put it into the namespace.

Reply:
Ok, then you might be better off putting it in an anonymous namespace